src/pages/VerifyEmail.tsx

```diff
@@ -1,5 +1,9 @@
 import { useEffect, useMemo, useState } from "react";
 import { useLocation } from "react-router-dom";
+import Background from "../assets/Background.jpg";
+import { Navbar } from "../components";
+import ErrorComponent from "../components/ErrorComponent";
```

Comment:
Add ErrorComponent and SuccessComponent to `index.ts` so it can be inline with the import on line 4